libraries/x509_helper.rb

```diff
@@ -0,0 +1,25 @@
+require 'openssl'
```

Comment:
probably want to stick these in the helpers so we don't have to do the require unless we actually use those. It'll speed up chef runs